### PR TITLE
Small improvements on `Style` docs

### PR DESCRIPTION
**Pull request by jush**
### Summary of changes

While reading documentation for [Style](https://docs.mapbox.com/android/maps/api/10.7.0/mapbox-maps-android/com.mapbox.maps/-style/), I noticed that the docs could use some cleaning.

### User impact (optional)

N/A

## Pull request checklist:
 - [ ] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Run `make update-api` to update generated api files, if there's public API changes, otherwise the `verify-api-*` CI steps might fail.
 - [ ] Update [CHANGELOG.md](../CHANGELOG.md) or use the label 'skip changelog', otherwise `check changelog` CI step will fail.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Review by kiryldz — APPROVED** [review]
Add `skip changelog` label? In general it's good practice to add labels as e.g. if you add `bug` label but forget to add changelog entry - CI will then fail.

File: sdk/src/main/java/com/mapbox/maps/Style.kt [hunk 1]
```diff
@@ -12,7 +12,7 @@ import java.nio.ByteBuffer
  * The general class to interact with Styles in the Modular Mapbox Maps SDK for Android.
  * It exposes the entry point for all methods related to the Style.
  * You cannot instantiate [Style] object directly, rather, you must obtain one
- * from the getStyleAsync() method on [MapboxMap].
```
Comment (pengdev) [inline sdk/src/main/java/com/mapbox/maps/Style.kt:15]: would be good to do an audit to all similar issues in the Style object, and make all the changes in a single PR

**Review by pengdev — APPROVED** [review]
LGTM with 1 comment

File: sdk/src/main/java/com/mapbox/maps/Style.kt (hunk 1 above)
Comment (jush) [reply sdk/src/main/java/com/mapbox/maps/Style.kt:15]: I added many more fixes. Please take a second look.

**Comment by kiryldz** [comment]
@jush perhaps update PR name a bit to better reflect to the scope of changes

**Pull request merged**